js/views/startScreen.js

```diff
@@ -1,155 +1,151 @@
-    }
+    //CSS class manipulation from config/skin.json
+    var stateScreenPosterClass = ClassNames({
+      'state-screen-poster': this.props.skinConfig.startScreen.promoImageSize != "small",
+      'state-screen-poster-small': this.props.skinConfig.startScreen.promoImageSize == "small"
+    });
+    var infoPanelClass = ClassNames({
+      'state-screen-info': true,
+      'info-panel-top': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("top") > -1,
+      'info-panel-bottom': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("bottom") > -1,
+      'info-panel-left': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("left") > -1,
+      'info-panel-right': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("right") > -1
+    });
+    var titleClass = ClassNames({
+      'state-screen-title': true,
+      'text-truncate': true,
+      'text-capitalize': true,
+      'pull-right': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("right") > -1
+    });
+    var descriptionClass = ClassNames({
+      'state-screen-description': true,
+      'pull-right': this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("right") > -1
+    });
+    var actionIconClass = ClassNames({
+      'action-icon': true,
+      'action-icon-top': this.props.skinConfig.startScreen.playButtonPosition.toLowerCase().indexOf("top") > -1,
+      'action-icon-bottom': this.props.skinConfig.startScreen.playButtonPosition.toLowerCase().indexOf("bottom") > -1,
+      'action-icon-left': this.props.skinConfig.startScreen.playButtonPosition.toLowerCase().indexOf("left") > -1,
+      'action-icon-right': this.props.skinConfig.startScreen.playButtonPosition.toLowerCase().indexOf("right") > -1,
+      'hidden': !this.props.skinConfig.startScreen.showPlayButton
+    });
 
-    if (this.props.skinConfig.startScreen.showTitle ||
-      this.props.skinConfig.startScreen.showDescription) {
-      if (this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("top") > -1)
-        infoStyle.style.top = "5%";
-      if (this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("bottom") > -1)
-        infoStyle.style.bottom = "5%";
-      if (this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("left") > -1)
-        infoStyle.style.left = "5%";
-      if (this.props.skinConfig.startScreen.infoPanelPosition.toLowerCase().indexOf("right") > -1) {
-        infoStyle.style.right = "5%";
-        infoStyle.title.style.float = "right";
-        infoStyle.description.style.float = "right";
-      }
-    }
+    var titleMetadata = (<div className={titleClass} style={titleStyle}>{this.props.contentTree.title}</div>);
+    var descriptionMetadata = (<div className={descriptionClass} style={descriptionStyle}>{this.props.contentTree.description.substring(0,90)}&hellip;</div>);
```

Comment:
This doesn't guarantee that the description maxes out at 2 lines with ellipses for the truncated text which was the ask from UI/UX.